src/core/connection.c

```diff
@@ -5532,7 +5762,7 @@ QuicConnParamSet(
         }
 
         QuicConnOnQuicVersionSet(Connection);
-        QuicConnProcessPeerTransportParameters(Connection, TRUE);
+        (void)QuicConnProcessPeerTransportParameters(Connection, TRUE);
```

Comment:
Do we really not care about the outcome of this function?

Reply:
Yeah, in the case where `FromResumption` is `TRUE`, none of the failure paths execute. I made this into a debug assert just in case to help future developers not break this path.